appveyor.yml

```diff
@@ -1,13 +1,5 @@
-version: 0.9.5.{build}
-branches:
-  only:
-  - master
-before_build:
-- nuget restore
-build:
-  project: build\build.proj
-  verbosity: normal
-test: off
-artifacts:
-- path: artifacts\*.nupkg
-  name: Nuget
\ No newline at end of file
+version: 1.0.{build}
+image: Visual Studio 2017
+build_script:
+- cmd: PowerShell -Version 2.0 .\build.ps1
```

Comment:
I've started trying to bootstap using build.cmd rather than depending on Powershell.  It seems to be the preferred convention.

Reply:
No problem